packages/@lwc/engine-core/src/framework/stylesheet.ts

```diff
@@ -221,7 +221,7 @@ export function createStylesheet(vm: VM, stylesheets: string[]): VNode | null {
                 insertGlobalStylesheet(stylesheets[i]);
             } else {
                 // local level
-                insertStylesheet(stylesheets[i], root!.cmpRoot);
+                insertStylesheet(stylesheets[i], root!.shadowRoot);
```

Comment:
why not just do `root!.renderRoot` and remove `shadowRoot` entirely from the vm?

Reply:
I don't think it's possible to get rid of the `shadowRoot` property as is also used in the `get LightningElement.prototype.template`.